angular.json

```diff
@@ -154,6 +154,123 @@
+                }
+              ],
+              "optimization": true,
+              "outputHashing": "all",
+              "sourceMap": false,
+              "extractCss": true,
+              "namedChunks": false,
+              "aot": true,
+              "extractLicenses": true,
+              "vendorChunk": false,
+              "buildOptimizer": true,
+              "budgets": [
+                {
+                  "type": "initial",
+                  "maximumWarning": "2mb",
+                  "maximumError": "5mb"
+                }
+              ]
+            }
+          }
+        },
+        "serve": {
+          "builder": "@angular-devkit/build-angular:dev-server",
+          "options": {
+            "browserTarget": "ngx-scrollbar-strict-null-checks:build"
+          },
+          "configurations": {
+            "production": {
+              "browserTarget": "ngx-scrollbar-strict-null-checks:build:production"
+            }
+          }
+        },
+        "extract-i18n": {
+          "builder": "@angular-devkit/build-angular:extract-i18n",
+          "options": {
+            "browserTarget": "ngx-scrollbar-strict-null-checks:build"
+          }
+        },
+        "test": {
+          "builder": "@angular-devkit/build-angular:karma",
+          "options": {
+            "main": "projects/ngx-scrollbar-strict-null-checks/src/test.ts",
+            "polyfills": "projects/ngx-scrollbar-strict-null-checks/src/polyfills.ts",
+            "tsConfig": "projects/ngx-scrollbar-strict-null-checks/tsconfig.spec.json",
+            "karmaConfig": "projects/ngx-scrollbar-strict-null-checks/karma.conf.js",
+            "assets": [
+              "projects/ngx-scrollbar-strict-null-checks/src/favicon.ico",
+              "projects/ngx-scrollbar-strict-null-checks/src/assets"
+            ],
+            "styles": [
+              "projects/ngx-scrollbar-strict-null-checks/src/styles.css"
+            ],
+            "scripts": []
+          }
+        },
+        "lint": {
+          "builder": "@angular-devkit/build-angular:tslint",
+          "options": {
+            "tsConfig": [
+              "projects/ngx-scrollbar-strict-null-checks/tsconfig.app.json",
+              "projects/ngx-scrollbar-strict-null-checks/tsconfig.spec.json",
+              "projects/ngx-scrollbar-strict-null-checks/e2e/tsconfig.json"
+            ],
+            "exclude": [
+              "**/node_modules/**"
+            ]
+          }
+        },
+        "e2e": {
+          "builder": "@angular-devkit/build-angular:protractor",
+          "options": {
+            "protractorConfig": "projects/ngx-scrollbar-strict-null-checks/e2e/protractor.conf.js",
+            "devServerTarget": "ngx-scrollbar-strict-null-checks:serve"
+          },
+          "configurations": {
+            "production": {
+              "devServerTarget": "ngx-scrollbar-strict-null-checks:serve:production"
+            }
+          }
+        }
+      }
```

Comment:
Why a new project?

Reply:
It was the simplest for me to reproduce the issue that I had while trying to fix it in context